src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -532,8 +532,8 @@ def list_sps(client, spn=None, display_name=None, query_filter=None):
 
     if not sub_filters:
         logger.warning('In a future relase, if no filter arguments are provided, CLI will only output the first'
-        ' 100 objects to avoid long waiting. You can still use "--all" for old behavior, though it is much'
-        ' recommended to avoid it')
+                       ' 100 objects to avoid long waiting. You can still use "--all" for old behavior,'
+                       ' though it is much recommended to avoid it')
```

Comment:
Recommended wording:
"In a future release, if no filter arguments are provided, CLI will output only the first 100 objects to minimize wait times. You can still use `--all` for the old behavior, though it is not recommended."